.github/workflows/ci.yaml

```diff
@@ -24,19 +28,19 @@ jobs:
       matrix:
         os: ['ubuntu',]
         python-version:
-          - "3.9"
           - "3.10"
           - "3.11"
+          - "3.12"
         include:
           - os: 'macos'
-            python-version: "3.11"
+            python-version: "3.12"
 
     steps:
       - uses: actions/checkout@v4
       - name: Get current date
         id: date
         run: echo "date=$(date +%Y-%m-%d)" >> "${GITHUB_OUTPUT}"
-      - uses: mamba-org/setup-micromamba@v1
+      - uses: mamba-org/setup-micromamba@v2
```

Comment:
we're still using micromamba v1 for openfe - do we want to bump that to v2? (personally /coI would prefer to)

Reply:
We should - it's not going to break today, but it's always nice to do it.